flexget/plugins/clients/rtorrent.py

```diff
@@ -637,8 +704,16 @@ def add_entry(self, client, entry, options, start=True, mkdir=False, fast_resume
             # No existing found
             pass
 
+        # Setup options and custom fields
+        if 'custom_fields' in options:
+            custom_fields = options.pop('custom_fields')
+        else:
+            custom_fields = {}
```

Comment:
`pop` can take a default to make this a bit simpler.
```suggestion
        custom_fields = options.pop('custom_fields', {})
```